src/TheThingsNetwork.cpp

```diff
@@ -969,11 +1007,9 @@ void TheThingsNetwork::sleep(uint32_t mseconds)
   modemStream->write(buffer);
   modemStream->write(SEND_MSG);
   debugPrintLn(buffer);
-}
 
-void TheThingsNetwork::wake()
-{ 
-  autoBaud();
+  // to be determined back on wake up
+  baudDetermined = false;
```

Comment:
Again, `moduleSleeping` would make more sense here.

Reply:
that's fine